src/middlewared/middlewared/plugins/network.py

```diff
@@ -1331,6 +1342,41 @@ def __convert_aliases_to_datastore(self, data):
                     )
             raise
 
+        if new.get('disable_offload_capabilities') != iface.get('disable_offload_capabilities'):
+            failover_licensed = not await self.middleware.call('system.is_freenas') and await self.middleware.call(
+                'failover.licensed'
+            )
+            if new['disable_offload_capabilities']:
+                await self.middleware.call('interface.disable_capabilities', iface['name'])
+                if failover_licensed:
+                    try:
+                        await self.middleware.call(
+                            'failover.call_remote', 'interface.disable_capabilities', [iface['name']]
+                        )
+                    except Exception as e:
+                        self.middleware.logger.debug(
+                            f'Failed to disable capabilities for {iface["name"]} on remote node: {e}'
+                        )
+            else:
+                capabilities = await self.nic_capabilities()
+                if failover_licensed:
+                    nics = await self.to_disable_evil_nic_capabilities(False)
+                    capabilities = [
+                        c for c in capabilities
+                        if c not in nics.get(iface['name'], []) and c not in iface['state']['capabilities']
+                    ]
+                if capabilities:
+                    await self.middleware.call('interface.enable_capabilities', iface['name'], capabilities)
+                    if failover_licensed:
+                        try:
+                            await self.middleware.call(
+                                'failover.call_remote', 'interface.enable_capabilities', [iface['name'], capabilities]
+                            )
+                        except Exception as e:
+                            self.middleware.logger.debug(
+                                f'Failed to enable capabilities for {iface["name"]} on remote node: {e}'
```

Comment:
The `remote node` was removed as accepted verbiage for TrueNAS HA. Let's use `for {nic} on standby storage controller`, please.